docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -180,17 +197,17 @@ Update the following environment variables:
 > #### Database note
 > This example uses SQLite as a data store. In production, we recommend using a database that has more robust concurrency support, such as PostgreSQL. For information on switching databases, see the [Replacing SQLite](#replacing-sqlite) section.
 
-1. If you're using SQLite, which is Subscription Foundation's default data store, skip to the next step. Otherwise, see [Replacing SQLite](#replacing-sqlite)
+1. If you're using SQLite, which is the default data store for Subscription Foundation, skip to the next step. Otherwise, follow the instructions in [Replacing SQLite](#replacing-sqlite)
 
-2. Run the pre-configured Prisma migration to create the database tables and initial client as defined in `/prisma/migrations/*`. Troubleshoot as needed.
+2. Run the pre-configured Prisma migration script to create the database tables and initial client as defined in `/prisma/migrations/*` by issuing the following command:
```

Comment:
1. issuing again
2. the troubleshoot note is, I think, to encourage goodwill and perseverance. migrations rarely work the first time and tend to require a bunch of side work and other terminal commands.